zasilkovna.php

```diff
@@ -751,17 +783,23 @@ public function plgVmDisplayListFEShipment(VirtueMartCart $cart, $selected = 0,
                 continue;
             }
 
+            if ($isHdCarrier) {
+                $hdCarrier = $this->carrierRepository->getCarrierById($zasMethod->getHdCarrierId());
```

Comment:
Má být výstup z $zasMethod->getHdCarrierId() ošetřený? Všiml jsem si podmínky jinde, tak mě zajímá proč to zde není.